crypto/ed25519/ed25519.go

```diff
@@ -75,15 +63,13 @@ func (privKey PrivKeyEd25519) PubKey() crypto.PubKey {
 		panic("Expected PrivKeyEd25519 to include concatenated pubkey bytes")
 	}
 
-	var pubkeyBytes [PubKeyEd25519Size]byte
-	copy(pubkeyBytes[:], privKeyBytes[32:])
```

Comment:
not an expert in cryptography, but sometimes copying is necessary because you don't want to refer to existing Go slice, you want an actual copy (privKey[32:] is a pointer!)

Reply:
thank you, reverted these removals